create_vm.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+usage() {
+  cat <<EOF
+  usage: $0 [-r]
+  This script creates
+    1) an empty VM with CentOS 6.8 ready to be installed, or
+    2) CentOS 6.8 VM when -r parameter is applied (reuses vdi from vdis/).
+  OPTIONS:
+    -h Show this message
+    -r Reuses a vdi that already exists.
+EOF
+}
+
+create_vm() {
+  echo -e "Creating an empty vm..."
+  source scripts/create_empty_vm.sh
```

Comment:
why using source when what you want to do is launch this script?
If you want to run this in the same shell, why not export VM and OS_TYPE variables (from create_empty_vm.sh)?